moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -351,11 +351,19 @@ void ServoCalcs::calculateSingleIteration()
   else
   {
     // Joint trajectory is not populated with anything, so set it to the last positions and 0 velocity
-    *joint_trajectory = *last_sent_command_;
+    joint_trajectory->header = last_sent_command_->header;
+    trajectory_msgs::JointTrajectoryPoint point;
+    point.time_from_start = ros::Duration(parameters_.publish_period);
+
+    current_state_->copyJointGroupPositions(joint_model_group_, point.positions);
     for (auto& point : joint_trajectory->points)
     {
       point.velocities.assign(point.velocities.size(), 0);
+      point.accelerations.assign(point.accelerations.size(), 0);
     }
+
+    joint_trajectory->points.clear();
+    joint_trajectory->points.push_back(point);
```

Comment:
I'd suggest re-using the point from `last_send_command_` like this:
I'd suggest re-using the point from `last_send_command_` like this:
I'd suggest to re-use the `points` from `last_sent_command_` like this. (Please test!)

```suggestion
    joint_trajectory = last_sent_command_;
    for (auto& point : joint_trajectory->points)
    {
      point.velocities.assign(point.velocities.size(), 0);
      point.accelerations.assign(point.accelerations.size(), 0);
    }
```

Reply:
I tried it it is fine, just that pointers were necessary (as in the original version) to make it to compile:
```
    *joint_trajectory = *last_sent_command_;
```